config.inc.php

```diff
@@ -119,6 +119,14 @@ function language_hook($PALANG, $language) {
 // Leave blank to send email from the logged-in Admin's Email address.
 $CONF['admin_email'] = '';
 
+// Site admin name
+// This will be used as signature in notification messages
+$CONF['admin_name'] = 'Postmaster';
+
+// Site admin phone number
+// This will be used if a user cannot access his/her email and needs support
+$CONF['admin_phone'] = '';
+
```

Comment:
Indeed admin_phone was intended for letting the user know how to contact the postmaster if his account doesn't have alternative contact detail, but is not used anymore, I've removed it.